iam_policies.tf

```diff
@@ -192,6 +192,14 @@ data "aws_iam_policy_document" "shepherd_users_other" {
     }
   }
 
+  statement {
+    effect = "Allow"
+    actions = [
+      "cloudtrail:*",
```

Comment:
Do shepherd users need access to cloudtrail?

Reply:
Ahh they do not. I pushed that for testing purposes earlier. I'll remove that and push changes.